core/src/main/java/com/linecorp/armeria/common/metric/MoreMeters.java

```diff
@@ -160,16 +159,32 @@ public static Timer newTimer(MeterRegistry registry, String name, Iterable<Tag>
         final Duration minExpectedValue =
                 minExpectedValueNanos != null ? Duration.ofNanos(minExpectedValueNanos.longValue()) : null;
 
-        return Timer.builder(name)
-                    .tags(tags)
-                    .maximumExpectedValue(maxExpectedValue)
-                    .minimumExpectedValue(minExpectedValue)
-                    .publishPercentiles(distStatsConfig.getPercentiles())
-                    .publishPercentileHistogram(distStatsConfig.isPercentileHistogram())
-                    .percentilePrecision(distStatsConfig.getPercentilePrecision())
-                    .distributionStatisticBufferLength(distStatsConfig.getBufferLength())
-                    .distributionStatisticExpiry(distStatsConfig.getExpiry())
-                    .register(registry);
+        final Timer.Builder builder =
+                Timer.builder(name)
+                     .tags(tags)
+                     .maximumExpectedValue(maxExpectedValue)
+                     .minimumExpectedValue(minExpectedValue)
+                     .publishPercentiles(distStatsConfig.getPercentiles())
```

Comment:
Q) Are there any needs to use different configurations for timer and distribution summary? 

It is unrelated to this PR, but I was wondering if it was a good idea to provide an API to allow users to customize each meter.
For example:
```java
interface MeterFactory {
  Timer newTimer(MeterRegistry registry, String name, Iterable<Tag> tags);

  DistributionSummary newDistributionSummary(MeterRegistry registry,
                                             String name, Iterable<Tag> tags);
}
```

Reply:
It would be nice if we can provide different config for `distributionSummary` and `timer`.
`distributionSummary` appears to be used to measure `content-length` in Armeria, and adding custom config for timer can potentially make histogram metrics for `content-length` useless.

Reply:
That is correct. Let me create an issue for that.

Reply:
Created: https://github.com/line/armeria/issues/5665